src/observables/FromEventPatternObservable.ts

```diff
@@ -0,0 +1,72 @@
+import Scheduler from '../Scheduler';
+import Observable from '../Observable';
+import Subscription from '../Subscription';
+import tryCatch from '../util/tryCatch';
+import {errorObject} from '../util/errorObject';
+
+export default class FromEventPatternObservable<T, R> extends Observable<T> {
+
+  static create<T, R>(addHandler: (handler: Function) => void, removeHandler: (handler: Function) => void, selector?: (x: R) => T, scheduler?: Scheduler) {
+    return new FromEventPatternObservable(addHandler, removeHandler, selector, scheduler);;
+  }
+  
+  static dispatch(state) {
+    const { addHandler, removeHandler, selector, subscriber } = state;
+    let scheduler = (<any>this).scheduler;
+    
+    let handler = selector ? (e) => {
```

Comment:
You should assign e to arguments[0] and a selector if there are more args should work, like we have in our existing RxJS [`fromEventPattern`](https://github.com/Reactive-Extensions/RxJS/blob/master/src/core/linq/observable/fromeventpattern.js#L11-L16)